prep-html-skeleton/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Skeleton Exercise</title>
+    <body>
+      <h1>I built a proper HTML skeleton!</h1>
+    </body>
```

Comment:
Hi Dan, be sure to make your `<body>` element a sibling of the `<head>` element instead of a child.